src/watchmaker/static/shell/bootstrappers/watchmaker-bootstrap.ps1

```diff
@@ -57,6 +57,23 @@ function Install-Python-MSI {
   Start-Process "msiexec.exe" -ArgumentList ${Arguments} -Wait
 }
 
+function Install-Python-Packages {
```

Comment:
Since these installs are so similar, I kind of had in mind a generic function for installing python packages, rather than hard code the dependencies... The function could take a param `[String[]]$Packages` and install them all... What do you think?